client/src/pages/people/Show.tsx

```diff
@@ -172,6 +173,12 @@ const PersonShow = ({ pageDispatchers }: PersonShowProps) => {
   const { loading, error, data, refetch } = API.useApiQuery(GQL_GET_PERSON, {
     uuid
   })
+  useEffect(() => {
+    if (data?.person) {
+      const person = new Person(data ? data.person : {})
+      setAttachments(person.attachments || [])
+    }
+  }, [data])
```

Comment:
Why not simply:
```javascript
  useEffect(() => {
    setAttachments(data?.person?.attachments || [])
  }, [data])
```
I would also suggest to put the `useEffect` after `usePageTitle`.